helpers/PropertyCache.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace oat\generis\Helper;
+
+class PropertyCache
+{
+    /**
+     * Clear property cached data
+     */
+    public static function clearCachedValuesByTriples(Iterator $triples): void
```

Comment:
missing namespace

Reply:
I propose to add use Iterator to line 4 (between namespace and class)
```
namespace oat\generis\Helper;

use Iterator;

class PropertyCache
```